operator/deploy/operator.yaml

```diff
@@ -1,25 +1,15 @@
-###
-# TODOs:
-#  1) At somepoint this file should move to REPO_ROOT/deploy/
```

Comment:
That point is still valid, please don't remove it.

Reply:
Done. Added back the comment.